src/components/ContentMapping/ContentMapping.ts

```diff
@@ -22,7 +24,8 @@ type ContentMappingType = {
 export enum WidgetTypes {
     PLAIN_TEXT = "PLAIN_TEXT",
     STUB = "STUB",
-    EXAMPLE_IMAGE = "EXAMPLE_IMAGE"
+    EXAMPLE_IMAGE = "EXAMPLE_IMAGE",
+    WYSIWYG_EDITOR = "WYSIWYG"
```

Comment:
this has to be `WYSIWYG_EDITOR = "WYSIWYG_EDITOR"` otherwise parts of the code will crash.